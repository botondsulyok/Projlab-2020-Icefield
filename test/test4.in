field spawn 0
field dry 0
field wet 5 true 0
neighbours spawn dry
neighbours dry wet
entity esk1 Eskimo spawn
stats esk1
move esk1 dry
stats esk1
move esk1 wet
stats esk1