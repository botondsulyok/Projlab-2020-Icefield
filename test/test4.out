id: esk1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk1
type: eskimo
iceUnit: dry
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk1
type: eskimo
iceUnit: dry
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

